client/lib/paths/login/index.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Internal dependencies
+ */
+import { addQueryArgs } from 'lib/url';
+import config, { isEnabled } from 'config';
+
+export function login( { redirectTo } ) {
+	const url = isEnabled( 'wp-login' ) ? '/login' : config( 'login_url' );
```

Comment:
I think it should work with Desktop until we get rid of `wp-login` flag completely and update `login_url` to `/login` in the corresponding config files.